mapeditor/inspector/inspector.cpp

```diff
@@ -335,6 +335,15 @@ void Inspector::updateProperties(CGHeroInstance * o)
 		}
 		addProperty("Hero type", o->getHeroTypeID().hasValue() ? o->getHeroType()->getNameTranslated() : "", delegate, false);
 	}
+	{
+		const int maxRadius = 60;
+		auto * patrolDelegate = new InspectorDelegate;
+		patrolDelegate->options = { {QObject::tr("No patrol"), QVariant::fromValue(std::numeric_limits<ui32>::max())} };
+		for(int i = 0; i <= maxRadius; ++i)
+			patrolDelegate->options.push_back({ QObject::tr("%1 tile(s)").arg(i), QVariant::fromValue(i) });
+		auto patrolRadiusText = o->patrol.patrolling ? QObject::tr("%1 tile(s)").arg(o->patrol.patrolRadius) : QObject::tr("No patrol");
```

Comment:
this is an incorrect way to localise plurals. You should pass the number as third parameter to `tr()`, see here: https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals